### PR TITLE
1.9 dev

### PR DESCRIPTION
**Pull request by sanderland**
Features:
* engine failure help
* game report
* katago 1.8.2
* keyboard override theming
* mistake sounds+theming
* sound fix?

- [x] korean
- [x] simplified chinese
- [x] traditional chinese
- [x] japanese
- [x] german
- [x] french
- [x] russian

### CODE REVIEW

**Comment by sanderland** [comment]
@Harleqin @Tony-Liou @kaorahi @Dontbtme @Satoshi-Yoda this should be ready for translation now.

image
image

**Comment by kaorahi** [comment]
a9aa121 doesn't fix the font problem.

~~~
$ git show --quiet --pretty=format:'%h%d'
8c29b06 (HEAD -> 1.9, upstream/1.9)
$ git diff --stat
 katrain/i18n/locales/jp/LC_MESSAGES/katrain.mo | Bin 21300 -> 21414 bytes
 1 file changed, 0 insertions(+), 0 deletions(-)
~~~
font

**Comment by sanderland** [comment]
9d1c500 does

**Comment by kaorahi** [comment]
yes. fixed in 9d1c500. thx!

File: katrain/i18n/locales/tw/LC_MESSAGES/katrain.po [hunk 1]
```diff
@@ -900,7 +900,7 @@ msgstr "KataGo設定指南"
 
 #. label in little popup for analyzing entire game
 msgid "reanalyze mistakes only"
-msgstr "限制僅分析錯誤"
+msgstr "僅分析错误操作"
```
Comment (Tony-Liou) [inline katrain/i18n/locales/tw/LC_MESSAGES/katrain.po:903]: Simplified to Traditional
`错误` -> `錯誤`

**Review by Tony-Liou — CHANGES_REQUESTED** [review]
Traditional Chinese is used as writing system rather than Simplified in Taiwan.

File: katrain/i18n/locales/tw/LC_MESSAGES/katrain.po (hunk 1 above)
Comment (sanderland) [reply katrain/i18n/locales/tw/LC_MESSAGES/katrain.po:903]: @chingyulin is Taiwanese, you can fight here and agree on the best translation ;)

Comment (chingyulin) [reply katrain/i18n/locales/tw/LC_MESSAGES/katrain.po:903]: agree with @Tony-Liou on 錯誤, my mistake.  `僅分析錯誤操作`

**Pull request merged**